salt/modules/x509_v2.py

```diff
@@ -1374,7 +1374,7 @@ def expires(certificate, days=0):
     """
     cert = x509util.load_cert(certificate)
     # dates are encoded in UTC/GMT, they are returned as a naive datetime object
-    return cert.not_valid_after <= datetime.datetime.utcnow() + datetime.timedelta(
```

Comment:
This will fail because `cryptography` returns naive datetimes. They should be converted to UTC. Once there is a release with https://github.com/pyca/cryptography/issues/9186, this should be updated to try to use the new endpoints, otherwise fallback to the old one with the mentioned fix.